usefulUtilities/loudApp/appResources/loudApp_app.js

```diff
@@ -0,0 +1,559 @@
+//
+//  loudApp_app.js
+//
+//  Created by Robin Wilson and Milad Nazeri 8/20/2018
+//  Updated by Robin Wilson 3/26/2019
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+(function () {
+
+    var AppUi = Script.require("appUi");
+
+    // Data management
+    var userStore = {}; // houses all users, see User constructor for structure
+    // sent to Vue to update our html UI
+    // app options are housed here
+    var settings = {
+        users: [],
+        ui: {
+            isExpandingAudioEnabled: false,
+            searchWholeDomainForLoudestEnabled: false,
+            isListening: false
+        }
+    };
+    var DEBUG = true;
+
+    // #region AUDIO
+
+    // Updates avatar's audio level in the userStore
+    var AVERAGING_RATIO = 0.05,
+        LOUDNESS_FLOOR = 11.0,
+        LOUDNESS_SCALE = 2.8 / 5.0,
+        LOG2 = Math.log(2.0),
+        AUDIO_PEAK_DECAY = 0.02;
+    function updateAudioForAvatar(uuid) {
```

Comment:
why does this function not take `UPDATE_INTERVAL_TIME` into consideration? (it might be that it doesn't need to, i just don't yet understand)

Reply:
Doesn't need to reference UPDATE_INTERVAL_TIME. Since that would be set on startInterval

Reply:
Actually you're right. That would make sense to factor in